react/README.md

```diff
@@ -701,6 +702,15 @@ We don’t recommend using indexes for keys if the order of items may change.
   > Why? [`isMounted` is an anti-pattern][anti-pattern], is not available when using ES6 classes, and is on its way to being officially deprecated.
 
   [anti-pattern]: https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html
+  
+## Things to avoid
+
+Using deprecated APIs will create bad legacy code. Hence it's good to know that which APIs have been deprecated.
+
+These Component lifecycles are been deprecated since React version 16.3+ :
```

Comment:
Could we add, for each one, which methods should be used instead (with links to their documentation)?